idptest/tests/test_salesforce.py

```diff
@@ -4,7 +4,7 @@
 # standard library imports:
```

Comment:
I'll make a new commit that adds `absolute_imports, print_function, unicode_literals` to all the files. That covers all the __future__ imports for Python 2 except division.